app/views/accounts/_transaction.html.slim

```diff
@@ -1,8 +1,6 @@
 tr
   td.for-desktop= transaction.date
   td.for-mobile.no-wrap= transaction.short_date
-  td.for-desktop= transaction.from_account.name || transaction.from_account.email
-  td.for-desktop= transaction.to_account.name
```

Comment:
let's rephrase [app/views/transactions_mailer/transaction_notification.html.erb](https://github.com/widefix/pocketmoney/blob/9702ec530f61a80fe9d553977c4fa714a149d2c9/app/views/transactions_mailer/transaction_notification.html.erb#L4) as well. It should mention from and to